Content/SAFE.App.sln

```diff
@@ -3,11 +3,11 @@ Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio 15
 VisualStudioVersion = 15.0.27004.2005
 MinimumVisualStudioVersion = 10.0.40219.1
-Project("{6EC3EE1D-3C4E-46DD-8F32-0CC8E7565705}") = "Client", "src\Client\Client.fsproj", "{73E8E820-C8AA-47CC-BB2B-152CA4D0B855}"
+Project("{6EC3EE1D-3C4E-46DD-8F32-0CC8E7565705}") = "SAFE.App.Client", "src\SAFE.App.Client\SAFE.App.Client.fsproj", "{73E8E820-C8AA-47CC-BB2B-152CA4D0B855}"
 EndProject
-Project("{6EC3EE1D-3C4E-46DD-8F32-0CC8E7565705}") = "Server", "src\Server\Server.fsproj", "{73E8E820-C8AA-47CC-BB2B-152CA4D0B856}"
+Project("{6EC3EE1D-3C4E-46DD-8F32-0CC8E7565705}") = "SAFE.App.Server", "src\SAFE.App.Server\SAFE.App.Server.fsproj", "{73E8E820-C8AA-47CC-BB2B-152CA4D0B856}"
 EndProject
-Project("{F2A71F9B-5D33-465A-A702-920D77279786}") = "Shared", "src\Shared\Shared.fsproj", "{28F090FB-E75F-4BE9-9E09-A70B1DA02C88}"
+Project("{F2A71F9B-5D33-465A-A702-920D77279786}") = "SAFE.App.Shared", "src\SAFE.App.Shared\SAFE.App.Shared.fsproj", "{28F090FB-E75F-4BE9-9E09-A70B1DA02C88}"
```

Comment:
let's also reorder the projects in solution file: 
1) Shared
2) Server
3) Client
that order makes IMO most logical sense